modules/bbp-experiment/src/test/resources/data/bbp/experiment/subject/v0.1.0/min-fields.json

```diff
@@ -1,6 +1,5 @@
 {
     "@context": [
-        "{{base}}/contexts/nexus/shacl/rec-shacl-20170720/v0.1.0",
         "{{base}}/contexts/nexus/prov/rec-prov-o-20130430/v0.1.0",
```

Comment:
Do we need {{base}}/contexts/nexus/prov/rec-prov-o-20130430/v0.1.0 context in an entity ?